testsuite/run_sets/secondary_parallelizable.yml

```diff
@@ -78,5 +78,6 @@
 - features/secondary/min_check_patches_install.feature
 
 - features/secondary/minkvm_guests.feature
+- features/secondary/min_salt_migration.feature
```

Comment:
Double check that we can run this in parallel with other features without colliding please.

Reply:
Thanks for this. I initially thought I put it in `secondary.yml` since it depends on the default KVM test which sets up the `virthost` vor virtualization.

Reply:
It depends only on `minkvm_guests.feature` which sets up the virthost. Other than that it only relies on features from core and reposync. So I am not sure if I should put it in the normale `secondary.yml` file. I did not found out how and if we can mark a feature to be dependent of another feature so it will always run after that dependent feauture.

Reply:
The only way to support this, is to use the secondary.yml file where the features always run sequentially, but in the parallel yml file (even if now it's not running in parallel and therefore running sequentially, the idea is to don't have any special order)

Reply:
Ok, then I will move it. But I wonder how can we guarantee that the normal KVM test is executed before the Salt bundle migration test?